### ISSUE TITLE
Kotlin 1.5 support

### ISSUE THREAD

**Issue by joreilly** [kuuuurt/multiplatform-paging#13]
Hi, are there plans to create version that supports Kotlin 1.5.10 and related dependencies?

**Comment by kuuuurt** [comment]
Hi. It took a while as I found it hard to find some time to work on my own projects. I just released 0.4.0. It should be available in a few.

**Comment by joreilly** [comment]
@kuuuurt after updating to 0.4.0 (had been using 0.3.7 still) I'm getting error in iOS code that `hasNextPage` isn't resolving....looks from code that this was made private? Is there alternative way of getting that info now?

**Comment by kuuuurt** [comment]
Right, I forgot that that was from your use case before. It was part of the effort to remove `CommonFlow` from the paging library and leave the `Flow` consumption to the developer. Will make the variable public. Do you need it as a `Flow` or could it work as a normal variable?

**Comment by joreilly** [comment]
Normal variable still should be fine

**Comment by joreilly** [comment]
hmm, sorry....forgot that this was flow before....this is the iOS code I had fwiw

```
        repository.characterPager.hasNextPage.watch { nullableNextPage in
            if let hasNextPage = nullableNextPage {
                self.hasNextPage = hasNextPage.boolValue
            }
        }
```

If it had to be normal variable I guess I could check it at point that I get updated data....

**Comment by kuuuurt** [comment]
I just released 0.4.1 which should make `hasNextPage` visible

**Comment by joreilly** [comment]
That worked, thanks!
